addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -874,4 +874,13 @@ describe("links with inline image", () => {
             `<p>ab<a href="#">c</a>]d<img src="${base64Img}">exxf<img src="${base64Img}">g[<a href="#">h</a>i</p>`
         );
     });
+    test("when link image the toolbar should disapear and only the linkpopover should be visible", async () => {
```

Comment:
Can you add more to the flow here to test the exact expected behavior which means :

- Select the image
- Toolbar is open and make sure it's the image toolbar and not the text one
- Click the link button
- Make sure link dialog open and toolbar closed
- add some text in the url
- Apply
- make sure the dialog is closed, toolbar closed and link popover is open
- Close it
- Check that the content has changed and now we have an a element
- Select the image again
- Click on the link button
- Make sure that popover is open and not link dialog